app/src/main/java/app/insti/fragment/AboutFragment.java

```diff
@@ -66,6 +66,7 @@ public void onStart() {
             put(R.id.ranveerimg, "ranveer.jpg");
             put(R.id.amangourimg, "amangour.jpg");
             put(R.id.wnccimg, "wncc.jpg");
+            put(R.id.safwanimg, "mayu.jpg");
```

Comment:
Put your own image here

Reply:
Change this to something else and email me the pic you want to put there.